docs/callbacks.mdx

```diff
@@ -62,6 +71,16 @@ Content-Type: application/json
 }
 ```
 
+Only the `url` key is required in the Request. All others are optional.
+ - `url` - this represents the address that will be utilized to send a POST request, containing detailed information about the model's status
+ - `filter` - this is used to restrict the triggering of callbacks:
+   - `model_name` - python-style regular expression that aids in filtering the names of models that trigger a callback
```

Comment:
A minor update:
let's say `python-style regular expression used to filter the names of models that trigger a callback` (like it is for projects.